### ISSUE TITLE
init_pyrebloom will return before select a redis db

### ISSUE THREAD

**Issue by xkiracn** [seomoz/pyreBloom#21]
Hi
Thanks for your work =) However, I noticed pyrebloom cannot select redis db correctly sometimes.

In bloom.c, init_pyrebloom() will return before select a redis db while it does a success redis authentication.

**Comment by dlecocq** [comment]
That seems like the intended behavior. Are you getting a client back that has the wrong db selected?

**Comment by xkiracn** [comment]
Yes, set redis authentication required and select db > 0. In this situation, pyrebloom selects db 0 in fact.
